cfgov/jinja2/v1/_includes/molecules/social-media.html

```diff
@@ -0,0 +1,137 @@
+                </a>
+            </li>
+            <li class="list_item">
+                <a class="m-social-media_icon"
+                   href="https://www.linkedin.com/shareArticle?mini=true&url={{ request.url | urlencode }}&title={{ title }}%20Developer%20Network"
+                   target="_blank">
+                    <span class="cf-icon cf-icon-linkedin-square"></span>
+                    <span class="u-visually-hidden">Share on LinkedIn</span>
+                </a>
+            </li>
+            <li class="list_item">
+                <a class="m-social-media_icon"
+                   href="mailto:?subject={{ title }}&amp;body=Check out this page from the CFPB - {{ request.url | urlencode }}">
+                    <span class="cf-icon cf-icon-email-social-square"></span>
+                    <span class="u-visually-hidden">Share by email</span>
+                </a>
+            </li>
+        {% else %}
+            <li class="list_item">
+                <a class="m-social-media_icon"
+                   href="https://facebook.com/cfpb">
+                    <span class="cf-icon
+                                 cf-icon-facebook-square"></span>
+                    <span class="u-visually-hidden">Visit us on Facebook</span>
+                </a>
+            </li>
+            <li class="list_item">
+                <a class="m-social-media_icon"
+                   href="https://twitter.com/cfpb">
+                    <span class="cf-icon
+                                 cf-icon-twitter-square"></span>
+                    <span class="u-visually-hidden">Visit us on Twitter</span>
+                </a>
+            </li>
+            <li class="list_item">
+                <a class="m-social-media_icon"
+                   href="https://www.linkedin.com/company/consumer-financial-protection-bureau">
+                    <span class="cf-icon
+                                 cf-icon-linkedin-square"></span>
+                    <span class="u-visually-hidden">Visit us on LinkedIn</span>
+                </a>
+            </li>
+            <li class="list_item">
+                <a class="m-social-media_icon"
+                   href="https://www.youtube.com/user/cfpbvideo">
+                    <span class="cf-icon
+                                 cf-icon-youtube-square"></span>
+                    <span class="u-visually-hidden">Visit us on YouTube</span>
+                </a>
+            </li>
+            <li class="list_item">
+                <a class="m-social-media_icon"
+                   href="https://www.flickr.com/photos/cfpbphotos">
+                <span class="cf-icon
+                             cf-icon-flickr-square"></span>
+                <span class="u-visually-hidden">Visit us on Flickr</span>
+              </a>
+            </li>
+        {% endif %}
+
+    </ul>
+</div>
+{% endmacro %}
+
+{% macro _share_twitter_url(options) %}
+    {% set _share_twitter_url = 'http://twitter.com/intent/tweet' %}
```

Comment:
Is this necessary here when it's also [here](https://github.com/cfpb/cfgov-refresh/pull/1213/files#diff-a4bd24623a20111e4edb25cc1cc74656L34)